deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,93 @@
+
+
+elasticsearch:
+  host: ""
+  user: ""
+  pass: ""
+
+cloudDefend:
+  enabled: false
+
+kubernetes: 
+  version: 1.51.0
+  enabled: false
+  namespace: "default"
+      
+  system:
+    metrics:
```

Comment:
Also does not have logs?

Reply:
this system referes to this one https://github.com/elastic/integrations/blob/main/packages/kubernetes/data_stream/system/manifest.yml which if I am not mistaken there are no logs for it